<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="colorSlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>359</number>
    </property>
    <property name="value">
     <number>35</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="colorWidget" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="colorSpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>53</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>359</number>
    </property>
    <property name="value">
     <number>70</number>
    </property>
   </widget>
   <widget class="QLabel" name="imageFrame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Color to track</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="moveCursorCheckBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Move Cursor</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>colorSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>colorSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>colorSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
